Makefile

```diff
@@ -40,14 +40,18 @@ get-mel:
 	cp $(MELREPO)/*.h oommf/app/oxs/local/
 	rm -rf $(MELREPO)
 
-travis-build: SHELL:=/bin/bash
-travis-build:
+in-docker: SHELL:=/bin/bash
+in-docker:
 	docker build --no-cache -t dockertestimage .
 	docker run -ti -d --name testcontainer dockertestimage
-	docker exec testcontainer make test-all
+	docker exec testcontainer make test-all-docker
 	docker stop testcontainer
 	docker rm testcontainer
 
-test-all:
+test-all-docker:
 	tclsh $(OOMMFTCL) boxsi +fg oommf/app/oxs/examples/stdprob3.mif -exitondone 1
 	tclsh $(OOMMFTCL) boxsi +fg oommf/app/oxs/examples/stdprob4.mif -exitondone 1
+
+test-all-on-ubuntu-latest:
```

Comment:
How about calling `test-all-on-ubuntu-latest` just `test-all` ?

Reply:
There is a difference between these two targets since we do not have OOMMFTCL envvar outside docker. Should we define OOMMFTCL in workflow file?

Reply:
Yes, that would be most elegant.